sources/packages/api/src/route-controllers/note/note.aest.controller.ts

```diff
@@ -35,13 +38,11 @@ export class NotesController extends BaseController {
    * @param noteType Note type(General|Restriction|System Actions|Program) which is passed to filter the notes.
    * @returns Student Notes.
    */
-  @Groups(UserGroups.AESTUser)
-  @AllowAuthorizedParty(AuthorizedParties.aest)
   @Get("/student/:studentId")
   async getStudentDetails(
     @Param("studentId") studentId: number,
     @Query("noteType") noteType: string,
-  ): Promise<NoteDTO[]> {
+  ): Promise<NoteAPIOutDTO[]> {
     const studentNotes = await this.studentService.getStudentNotes(
       studentId,
       noteType as NoteType,
```

Comment:
`noteType` must be validated in the query string. It can be declared as the enum type and use the `ParseEnumPipe`.

Reply:
pls review the other endpoints with `noteType` too